install/README.md

```diff
@@ -1,101 +1,156 @@
-# ONLY FOR FRESH INSTALL AT NOW FOR OTHERS -> upgrade to geotrek 2.18.6
+#### /!\ Only for fresh installations, for others please upgrade to geotrek 2.18.6 /!\
 
-# INSTALL DOCKER
+
+## INSTALL Docker and Docker-Compose
+Check your Linux distribution :
+```bash
+lsb_release -a 
+```
+Find the most adequate docker install in :
 https://docs.docker.com/install/linux/docker-ce/ubuntu/
 
-# INSTALL DOCKER COMPOSE
+And Docker-Compose :
 https://docs.docker.com/compose/install/#install-compose
 
-# CREATE a dedicated user
+## CREATE a dedicated user
+Do the following 3 commands :
+
+```bash
 useradd geotrek
 adduser geotrek sudo
 adduser geotrek docker
+```
 
+## MOVE your folder
 
-# move and rename this folder according your instance name
+Move and rename this folder according to your instance name
 
 ex : 
 
 ```bash
-cd ..
-mv install /srv/geotrek
-
+cd .. # If you are inside the folder install
+mv install /path/of/your/instance/geotrek
 ```
+*Later in this install /path/of/your/instance/geotrek is /srv/geotrek*
 
-# Fix rights and login with your user for all operations
+## FIX rights and LOGIN with your user for all operations
+```bash
 chown -R geotrek:geotrek /srv/geotrek
 su - geotrek
+```
 
-# COPY .env.dist TO .env
+# COPY environment of install
+
+Copy .env.dist to .env
 
 ```bash
 cd /srv/geotrek
-cp .env.dist .env
+cp ./install/.env.dist .env
 
 ```
 
-# SET environment variables in it
+## SET environment variables
 
+Open .env and complete it
 ```dotenv
 POSTGRES_USER=<your_personnal_database_user>
 POSTGRES_DB=<your_personnal_database_user>
 POSTGRES_PASSWORD=<your_personnal_database_password>
 DOMAIN_NAME=<your.geotrek.com>
 SECRET_KEY=<your_personnal_secret_key>
 ```
-
-# Using external database (make sure postgresql > 9.3 and postgis > 2.1)
-
-add these environment variables
-
+**If you use external database, make sure postgresql > 9.3
+and postgis > 2.1, add these environment variables**
 ```dotenv
 POSTGRES_HOST=<your_host_or_ip>
 POSTGRES_PORT=<your_port>
 ```
+**and comment postgresql section in docker-compose.yml**
+```python
 
-and comment postgresql section in docker-compose.yml
+```
+## CREATE the var folder
+```bash
+./install/entrypoint.sh
+cd ./var
+```
 
-# Edit your custom.py file
-# SET LANGUAGES / SRID / SPATIAL_EXTENT / DEFAULT_STRUCTURE_NAME at least
+## EDIT your custom.py file
+Set at least LANGUAGES / SRID / SPATIAL_EXTENT / DEFAULT_STRUCTURE_NAME:
 
-# INITIATE database
+*You need to use sudo*
+
+ex:
+
+```bash
+sudo vi custom.py
+sudo nano custom.py
+# ...
+```
+
+```python
+
+LANGUAGES = (
```

Comment:
c'est : MODELTRANSLATION_LANGUAGES = ('en', 'fr', 'it', 'es')
sous cette forme

LANGUAGES, ce sont les traductions disponibles pour l'interface de geotrek admin, pas pour le contenu traduit (modeltranslation)

Reply:
la custo client, c'est modeltranslation que çà concerne